Author: Ken Zhu <guigenz@qti.qualcomm.com>
Date:   Tue Jan 20 15:20:10 2026 -0800

    [networking] fix mcproxy script issue

    for any string with space needs double quote.

    Change-Id: I2b238c925c3de4e39b60bda711780c1079ad2325
    Signed-off-by: Ken Zhu <guigenz@qti.qualcomm.com>

diff -Nur a/mcproxy/src/proxy/receiver.cpp b/mcproxy/src/proxy/receiver.cpp
--- a/mcproxy/src/proxy/receiver.cpp	2026-01-21 11:22:22.760801617 -0800
+++ b/mcproxy/src/proxy/receiver.cpp	2026-01-21 15:45:35.193352150 -0800
@@ -119,7 +119,9 @@
         }
 
         m_data_lock.lock();
-        analyse_packet(&msg, info_size);
+        if (m_running) {
+		analyse_packet(&msg, info_size);
+	}
         m_data_lock.unlock();
     }
 }
